Lib/functools.py

```diff
@@ -234,25 +234,26 @@ def __ge__(self, other):
 
 _initial_missing = object()
 
-def reduce(function, sequence, initial=_initial_missing):
+def reduce(function, iterable, initial=_initial_missing):
```

Comment:
In theory, if anyone is using this pure python code and if they are using keyword arguments, this is a breaking change.

I recommend just leaving this alone.

Reply:
I suggested a signature change on the issue, not knowing of the Python version, because 'sequence' in the C version  is positional only, hence could not be passed by keyword.  I agree that this complicates matters and definitely should not be done in a backported doc-change only patch.